optee-utee/src/lib.rs

```diff
@@ -35,6 +35,7 @@ use core::panic::PanicInfo;
 use optee_utee_sys as raw;
 
 #[cfg(not(target_os = "optee"))]
+#[cfg(not(feature = "no_panic_handler"))]
```

Comment:
merge the two condition into `#[cfg(all(not(target_os = "optee"), not(feature = "no_panic_handler")))]`